### DISCUSSION TITLE
Displaying relation managers separately

### DISCUSSION THREAD

**Discussion by kokoshneta** [filamentphp/filament#6675] [Help] [answered]
There’s a [previous question](https://github.com/filamentphp/filament/discussions/3471) about having more control over relation managers (the ability to place relation managers _inside_ an edit form just like any other layout component would be fantastic), but there doesn’t seem to be much by way of a solution for that as yet.

This question is more limited in scope:

I love the simplicity of relation managers, but I hate that they show up in tabs – you miss them, and they take longer to load because it’s three or four extra AJAX calls to switch to a new tab.

You can move the entire edit form for the base record into a tab so everything is tabbed, but is there a way to do the opposite, that is, move relation managers **out of** tabs, to just show them in separate section, one underneath the other?

**Comment by danharrin** [comment, marked as answer]
1) They cannot currently be embedded inside forms, as there are browser limitations that stop us from nesting `<form>` elements (like action forms) inside another `<form>` (the edit form).
2) We might be able to get around this if Livewire v3 allows us to teleport HTML to other parts of the page.
3) If we didn't use AJAX calls to load relation managers only when they are required, the initial page load would slow down.
4) You don't need to put relation managers in tabs, if they are all wrapped inside 1 RelationGroup then they are not tabbed.

**Reply by Lukaaashek** [reply]
I checked the code and found no form element for edit form anymore there. Would it be possible now?

**Reply by danharrin** [reply]
https://github.com/filamentphp/filament/blob/3.x/packages/panels/resources/views/resources/pages/edit-record.blade.php#L9-L20

**Reply by Lukaaashek** [reply]
OK, thanks for reply. I searched for `<form>` element. Livewire is still a bit confusing for me.

**Reply by danharrin** [reply]
It's a Blade component that wraps it

**Reply by liamseys** [reply]
@danharrin Has there been any update on this? I’m trying to move the relationship manager field to a dedicated section on the form edit page. It would be great if this functionality is now supported. Any insights would be appreciated!

**Reply by kokoshneta** [reply]
@liamseys It’s been supported all along – just put them inside a `RelationGroup` and they’ll be drawn underneath one another instead of in tabs.

**Reply by leandrocfe** [reply]
> @danharrin Has there been any update on this? I’m trying to move the relationship manager field to a dedicated section on the form edit page. It would be great if this functionality is now supported. Any insights would be appreciated!

Do you mean, something like this?

https://x.com/leandrocfe/status/1864854632272420864?t=7NKcCAN458jkIOrMDsnFjw&s=19
